src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadata.cs

```diff
@@ -63,8 +63,9 @@ public PackageIdentity Identity
         public Uri LicenseUrl { get; private set; }
 
         [JsonProperty(PropertyName = JsonProperties.Owners)]
-        [JsonConverter(typeof(MetadataFieldConverter))]
-        public string Owners { get; private set; }
+        public IEnumerable<string> OwnersEnumerable { get; private set; }
+
+        public string Owners => OwnersEnumerable != null ? string.Join(", ", OwnersEnumerable.Where(s => !string.IsNullOrWhiteSpace(s))) : null;
```

Comment:
This computes every time it's accessed. 
Can we cached it after the first computation?